griptape/drivers/event_listener/griptape_cloud_event_listener_driver.py

```diff
@@ -25,12 +26,14 @@ class GriptapeCloudEventListenerDriver(BaseEventListenerDriver):
         default=Factory(lambda: os.getenv("GT_CLOUD_BASE_URL", "https://cloud.griptape.ai")),
         kw_only=True,
     )
-    api_key: str = field(default=Factory(lambda: os.getenv("GT_CLOUD_API_KEY")), kw_only=True)
+    api_key: Optional[str] = field(default=Factory(lambda: os.getenv("GT_CLOUD_API_KEY")), kw_only=True)
```

Comment:
Why should `api_key` be allowed to be `None`?

Reply:
Because `os.getenv("GT_CLOUD_API_KEY")` returns `str | None`. Should we instead change to `os.environ`?